aeron-driver/src/main/c/aeron_publication_image.c

```diff
@@ -386,6 +385,11 @@ void aeron_publication_image_track_rebuild(aeron_publication_image_t *image, int
             }
         }
 
+        if (INT64_MAX == min_sub_pos)
+        {
+            return;
+        }
```

Comment:
I guess it should be impossible to hit this case due to the aeron_driver_subscribable_has_working_positions check?

Reply:
Yes, it is a belt & braces.